activerecord/lib/active_record/railtie.rb

```diff
@@ -30,6 +30,7 @@ class Railtie < Rails::Railtie # :nodoc:
     config.active_record.use_schema_cache_dump = true
     config.active_record.check_schema_cache_dump_version = true
     config.active_record.maintain_test_schema = true
+    config.active_record.parallel_test_table_reset_method = :truncate
```

Comment:
@djmb Can you please also add documentation for the new option to `guides/source/configuring.md` in the [Configuring Active Record](https://guides.rubyonrails.org/configuring.html#configuring-active-record) section?